### ISSUE TITLE
Interviews

### ISSUE THREAD

**Issue by scottgullaksen** [deco3500-2019/Apple_Pen#1]
# The aim with research methods
To construct interviews/research methods we must look at general design considerations (accross key interaction design, CSCW, emotional design issues) which is compounded by manifistation and genre specific issues to understand what considerations we must identify and understand with our research methods.

# The design considerations
Below are some relevant considerations that might be helpful for constructing specific research methods, e.g. interviews. You can use the numbers for references in your questions.
### 8 Groupware Challenges (Jonathan Grudin)
1. Mismatch between work required & benefit received across the group 
2. Need to have “critical mass” for collective benefit to made clear
3.  Need to provide incentive for un-selfish individual contribution
4. Needs to understand particular social & political subtexts of workplace
5. Design for the way things actually work (rather than how they should)
6. Don’t forget the individual within the group
7. Not all the work that is carried out is done as a group
8. Evaluation of use and impact is more complex
9. The decision-maker is usually not the end-user
10. The adoption process & change management
### Our soultion uses Collaboration Transparency
11. Designed to sit on top of legacy systems
12. Multiple Single-user
13. Potential solution to groupware ‘critical mass’ issues
14. Strict WYSIWIS
15. Central architecture

### WYSIWIS (What You See Is What I See)
16. How do you represent individual action in a group system?
17. What actions do you represent?
18. Undo/Redo - system vs user perspective

### Synchronisation
19. Synchronous/asynchronous communication
20. Centralised/Replicated/Hybrid architectures
21. The order of some actions is not important, but in others it is significant
22. e.g. collaborative editing of a shared document 

### Awareness : Shared Feedback
23. Visibility of who is around, what is happening, and who is talking with whom

### Awareness : Shared Info Spaces
24. Centralised representation
25. All activity represented

### Awareness : Social Translucence
26. Making collective activity visible
27. Translucent, not transparent
28. Representing social presence of collaborators
29. “minimalist graphical visualizations of the presence and activities of people participating”
## Context
Context is any information that can be used to characterize the situation of an entity. An entity is a person, place or object that is considered relevant to the interaction between a user and an application, including the user and the application themselves.

#### Location
30. same network (cartesian?)
31. Mode: physical
#### Mobility 
32. level: mobile
33. relation to others: Personal, group, public
34. User binding: Personal, Group, Public
#### Population
35. Bodies: People, devices
36. Manifestation: Physical, virtual
## Privacy/Security
37. How personal data is used, represented, stored
38. How made known to users
39. All users follow same model, or not
40. Users represented as individuals, or part of the masses
### The device
41. Existing user-owned (laptops and phones)
42. Single device
43. Centralized (at one laptop)

### The Scale of the Interface
45. Change across scale
46. Change across types of interface
47. Interface is the point at which a user acts on the system

## Social
48. Which attributes is important when identifying yourself to/in a system? Why?
49. What is irrelevant? Why?
50. What decisions do you make when deciding your response? 

### Users need to be represented in the system
51. Online identities don’t necessarily represent in-person
52. Allows for anonymity, for redefining one-self
53. Advantageous in some situations
54. Can form relationships without usual assumptions and
55. restrictions of face-to-face communication
56. Decide what is important for defining/representing identity

###  Anti-social behaviour
57. What if users don't use the system as intended and deliberetly tries to ruin for others?

## Mobile 
58. Calm technology valued over constant disruption
59. Device as continuous companion
60. Interactions are user-driven : need for high relevance
61. Interactions can extend beyond the device & may not be limited by the capabilities of the device
70. Are ofen smaller parts of a larger goal
71. Able to provide services that redefine our social networks and the places we inhabit

## Mobile Social
72. Data - Push vs Pull 
73. Representation of the presence of information 
74. Authority to author (who, type, verification) 
75. Explicit vs implicit social engagement
76. Direct/indirect
77. In-tech, through-tech, around-tech
78. Social interaction through place 

## Pursuasive computing
79. Selection of behaviour & audience is key
80. Behaviour : simple, specific, achievable
81. Audience : receptive, desiring change
82. Identify the barriers to behaviour change: lack of motivation, ability, well-timed prompt
83. Identify best channel for intervention
84. Social dynamics vs social cues

## General things we would like to know about our users
85. From the feedback we got, we were told that we should look at existing solutions to see what we can adopt for easy prototyping. Initiate conversation with what there experiences with such solutions are and build upon that.

**Comment by scottgullaksen** [comment]
## Potential Interview Questions
Please feel free to add some. This is just a draft and will be edited later on!

 - Can you come to think of any type of technologies you have used in a lecture before?
 - (To help - suggest asking:) What about kahoot.it, Google doc, twitter, open questions, solving problems on the blackboard/powerpoint (either individually or in groups), powerpoint in general, blackboard lecture, forums, google survey, etc.
 - From the technologies used - and in general - Do you participate in them? If not, why? If you do, what motivates you to participate? (1, 2, 3)
- What do you think about them? Are they fun? Are they engaging? Do you think it helps you learn better? Why, why not?
 - In what way and to what extent do you feel that your participation gets paid off? Who do you think benefits from it? (1, 2)
 - Do you feel visible/valuable when participating? Do you think that is a good thing/appropriate? (6, 7)
- How easily are these technologies adopted in the lecture? E.g. time to set up, perform actions, and distribute results? Do you think they should be better? (10)
- To what extent do you think actions are represented in the system? How precise are they and do you feel that is appropriate? (16, 17) 
- Do you feel like there is enough room to correct your errors? (18)
- What do you think about the responsiveness in these systems? Is it important? Why? (19)
- How do you recognize the virtual presence of the others and their actions in class? Is the level visibility appropriate? How does that affect you? (23-29)
- In these systems how much do you need to reveal about yourself and what do you think of it? Is it necessary? (37-40, 48-50). What do you usually respond? 
- Do they allow for anonymity? What do you think about that? Does it help your interaction or demotivate it? (51 - 56)
- Do you bring laptops or phones or any other mobile devices for lectures? How much time do you spend on this kind of devices each during the lecture? Is the work your doing there relevant? (41, 42, 44)

**Comment by scottgullaksen** [comment]
The requirements we identified in the week 8 workshop
![Requirements](https://user-images.githubusercontent.com/25436721/65086278-8a79da80-d9f4-11e9-8ab9-ed55d6f4e06a.jpg)

We can use this to focus on what needs to be answered/specified with our research in order to generate ideas for our designs!

**Comment by Annanyuan** [comment]
**Interview**:
#### Lecture Learning Experience:
- Pro: 
   1. Can follow lecturer instead read the PPT themselves.
   2. Can communicate with classmates if there is some learning point not clear.
   3. Solve the problems they have got in class.
- Con:
   1. Hard to follow lecturer sometimes because of language problem.
   2. Teaching method is not always suitable for students.
   3. The content teaching in lecture is too theoretical to engage them to study actively.
   4. Reviewing recording at home can be more efficient.
#### Kahoot:
- Pro: 
   1. Whole class can participate.
   2. Can discuss together.
   3. Can express herself fully.
#### Expect:
- Improve participant in lecture.
- Better to have a competition like having top few rankings to make it like a game.
- Less teaching content in lecture and slow the pace.
